tests/php/Controllers/SideMenu/SideMenuController/SideMenuControllerTopActionTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+use App\Http\Controllers\SideMenu\SideMenuController;
+use Illuminate\Http\Request;
+
+/**
+ * SideMenuControllerクラスのtopActionのテストを行う.
+ *
+ * @SuppressWarnings(PHPMD.TooManyPublicMethods)
+ */
+class SideMenuControllerTopActionTest extends TestCase
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)